### PR TITLE
添加readme

### PR DESCRIPTION
**Pull request by jstzwj**
主要添加了简介，介绍了如何编译和运行

### CODE REVIEW

File: src/MineCase.Server.Grains/User/UserGrain.cs [hunk 1]
```diff
@@ -142,7 +143,15 @@ public Task SetName(string name)
             _name = name;
             return Task.CompletedTask;
         }
-
+        public Task<uint> GetProtocolVersion()
+        {
+            return Task.FromResult(_protocolVersion);
+        }
+        public Task SetProtocolVersion(uint ver)
```
Comment (sunnycase) [inline src/MineCase.Server.Grains/User/UserGrain.cs]: add empty line

File: src/MineCase.Server.Grains/Network/Login/LoginFlowGrain.cs [hunk 2]
```diff
@@ -42,5 +60,13 @@ class LoginFlowGrain : Grain, ILoginFlow
                 UUID = uuid.ToString()
             });
         }
+        private async Task SendLoginDisconnect(string reason)
```
Comment (sunnycase) [inline src/MineCase.Server.Grains/Network/Login/LoginFlowGrain.cs]: maybe a `Chat `object.

File: src/MineCase.Server.Interfaces/User/IUser.cs [hunk 3]
```diff
@@ -14,6 +14,8 @@ namespace MineCase.Server.User
     public interface IUser : IGrainWithGuidKey
     {
         Task SetName(string name);
+        Task<uint> GetProtocolVersion();
+        Task SetProtocolVersion(uint ver);
```
Comment (sunnycase) [inline src/MineCase.Server.Interfaces/User/IUser.cs]: ver -> version

File: src/MineCase.Server.Grains/Network/Login/LoginFlowGrain.cs [hunk 4]
```diff
@@ -12,6 +12,13 @@ namespace MineCase.Server.Network.Login
     class LoginFlowGrain : Grain, ILoginFlow
     {
         private bool _useAuthentication = false;
+        /*
```
Comment (sunnycase) [inline src/MineCase.Server.Grains/Network/Login/LoginFlowGrain.cs]: clean unused comment.

File: Readme.md [hunk 5]
```diff
@@ -1,2 +1,102 @@
...
+`MineCase` 是一个分布式的 `minecraft` 服务端应用。`MineCase` 采用 `.net core` 编写，使用 `Orleans` 框架。<br>
+`MineCase` 具有以下特性：
+
+* 跨平台
+* 分布式
+
+## 要求
+运行这个服务器之前，请保证你满足：
+
+* `.net` core sdk 1.1
```
Comment (sunnycase) [inline Readme.md]: `.net`  core -> `.net core`

File: Readme.md [hunk 6]
```diff
@@ -1,2 +1,102 @@
...
+`MineCase` is a distributed `minecraft` server application. It is written in `.net core`, and using the `Orleans` framework. <br>
+`MineCase` has the following features:
+
+* cross-platform
+* distributed
+
+## Required
+Before running this server, please make sure you meet:
+
+* `.net` core sdk 1.1
```
Comment (sunnycase) [inline Readme.md]: `.net` core -> `.net core`

File: Readme.md (hunk 5 above)
Comment (akemimadoka) [inline Readme.md]: sdk 1.1 -> sdk (version >=1.1)

File: Readme.md (hunk 6 above)
Comment (akemimadoka) [inline Readme.md]: sdk 1.1 -> sdk (version >= 1.1)

File: Readme.md (hunk 5 above)
Comment (sunnycase) [inline Readme.md]: not >= 1.1, exactly 1.1. or you can't run it sucessfully.

**Pull request merged**